docs/rfcs/0020-collection-containment-membership.md

```diff
@@ -0,0 +1,165 @@
+
+...then we say the `items` and `containedItems` have **diverged**. I don't think we need to explicitly track this though, just notice it when a Manifest or Collection gets saved.
+
+> [!NOTE]
+> The `containedItems` property is generated by the hierarchical DB relationship. And the `items` property is just another "JSON is King" thing to be respected and stored whether we understand all of it or not. As long as it's a valid list, we can still append to it when a new containment relationship is added.
+
+There is some redundancy in that Storage Collections have `items` and `containedItems` that assert the same set of resources, but there is still a useful difference - one is the public `items` and one is not.
+
+> [!NOTE]
+> I have a vague feeling that this might do away with the distinction between IIIF and Storage Collections...? Everything is a IIIF Collection and the two properties are in sync - until they aren't. `items` becomes just another JSON property that we persist, except that we attempt to non-destructively edit it when there is a containment change (addition or deletion via PUT, POST or DELETE of a resource).
+
+The `containedItems` property should use the thumbnail property stored in the DB to generate containment navigation UI in the portal (and other management tools that speak the Extras API). This is why the thumbnail in the DB is a direct URL of an image rather than a service reference. No need for a thumbnail image service:
+
+
+```json
+{
+   "containedItems": [
+      {
+         "id": "https://iiif.dlcs.io/7/manifests/abcd",
+         "type": "Manifest",
+         "label": { "en": [ "Fellowship"] },
+         "thumbnail": [ { "id": "https://dlcs.io/thumbs/7/99/fs/full/!100,100/0/default.jpg", "type": "Image" } ]
+      }
+   ]
+}
+```
+
+(The string "https://dlcs.io/thumbs/7/99/fs/full/!100,100/0/default.jpg" is what we stored in the DB for this Manifest, it just happens to be an Image Service URI). That's all it ever needs, I think, to drive repository-browsing UI. That thumb is the exact string from the DB row, no need to compute it or look it up. Public IIIF never sees this.
+
+For API callers who stick to vanilla IIIF only (no HTTP operations on flat API, pure vanilla IIIF payloads) this model _just works_ - although they never see a `containedItems` property, their `items` will be auto-generated as far as possible, and they can modify the JSON if they want to.
+
+
+
+Public view of Storage Collection `items` are always just expanded versions of the above (e.g., with image service on the thumbnail, maybe...) (???)
+
+Public view of IIIF Collections `items` are sometimes expanded versions of the above, and will be until you do something to change it, but could be absolutely any list of Manifests and Collections anywhere:
+
+ * things that are actually contained too (most common)
+ * external things
+ * things that are managed by IIIF-CS but are in a different location in the repository (member, but not contained)
```

Comment:
Do we need to differentiate this? Or is storing an external Id or something in the DB enough? What is the DB table that stores this? `canvasPaintings` but not